Gems/PhysX/Code/Tests/PhysXJointsTest.cpp

```diff
@@ -329,23 +329,23 @@ namespace PhysX
 
         if (auto* sceneInterface = AZ::Interface<AzPhysics::SceneInterface>::Get())
         {
-            jointHandle = sceneInterface->AddJoint(m_testSceneHandle, &jointConfiguration, m_parentBodyHandle, m_childBodyHandle);    
+            jointHandle = sceneInterface->AddJoint(GenericPhysicsFixture::m_testSceneHandle, &jointConfiguration, this->m_parentBodyHandle, this->m_childBodyHandle);
```

Comment:
If `this->` is going to be used for m_parentBodyHandle and m_childBodyHandle, then it should also be used for m_testSceneHandle for consistency.
```suggestion
            jointHandle = sceneInterface->AddJoint(this->m_testSceneHandle, &jointConfiguration, this->m_parentBodyHandle, this->m_childBodyHandle);
```

Reply:
Let me try, I think I tried it

Reply:
it worked, changed